recipe/conda_build_config.yaml

```diff
@@ -543,6 +543,9 @@ python:
   - 2.7
   - 3.6
   - 3.7
+  # pypy only supports win32 so far
+  - pypy.3.6.7.1  # [unix or win32]
+  - pypy.2.7.7.1  # [unix or win32]
```

Comment:
We need to build `pypy` with those version strings first, right?

Reply:
Sorry, I am a conda newbie. What exactly do I need to do to 
- "install conda-smithy from this PR and re-render a feedstock"

Is there a "getting started" page somewhere?

Reply:
Sorry I was not clear. You would need a development install of the package with this PR. I usually clone the repo, checkout the branch, and do `pip install -e . --no-deps --force-reinstall`. However, there are a lot of details I left out in order to build a feedstock locally. Some of those are listed in our docs: https://conda-forge.org/docs/maintainer/updating_pkgs.html?#testing-changes-locally

I guess that the best course of action is for us, `conda-forge`, to vote on this PR and merge it ASAP. I'll try to run a few tests myself this weekend and I'll get back to you.

Reply:
@ocefpaf did you have a chance to take a look?

Reply:
Sorry, no. Kind of swamped with the day job and some travels in the agenda. But I did not forget this and it is part of `conda-forge`'s plan to roll this out ASAP.

Reply:
ping